framework/src/meshgenerators/ElementGenerator.C

```diff
@@ -112,6 +114,11 @@ ElementGenerator::getElemType(const std::string & type)
     Elem * elem = new Tri6;
     return elem;
   }
+  if (type == "TRI7")
```

Comment:
Can we just replace this whole function with `return Elem::build(Utility::string_to_enum<ElemType>(type)).release()`?

Reply:
I'd personally change it to return `unique_ptr` too, but maybe in a separate commit, since that's a change to a public API and I wouldn't want to hold up this PR if some downstream app has started using it.